pkg/kube/controllers/quarksstatefulset/active_passive_reconciler.go

```diff
@@ -56,7 +56,7 @@ type ReconcileStatefulSetActivePassive struct {
 // container.
 // Note:
 // Reconcile will always requeue on success after ActivePassiveProbe.PeriodSeconds have passed.
-func (r *ReconcileStatefulSetActivePassive) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *ReconcileStatefulSetActivePassive) Reconcile(_ context.Context, request reconcile.Request) (reconcile.Result, error) {
```

Comment:
I wonder if that's better than the context we store one the struct.

Reply:
Also, we would like to use the parent context to create the new context instead of the one from the argument.